client/state/test/index.js

```diff
@@ -0,0 +1,48 @@
+require( 'lib/react-test-env-setup' )();
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import sinon from 'sinon';
+
+describe( 'state', () => {
+	var createReduxStore;
+	before( () => {
+		createReduxStore = require( 'state' ).createReduxStore;
```

Comment:
Why did you need to place this in `before`, as opposed to importing at top-of-file?